src/ca_certs.pem

```diff
@@ -0,0 +1,239 @@
+subject= /C=US/O=DigiCert Inc/OU=www.digicert.com/CN=DigiCert Assured ID Root CA
```

Comment:
Is this the default duo ca cert? Like the one in authproxy repo?

Reply:
It's the same one from `duo_client_python`, that's where this was pulled from